internal/apps/ai-proxy/common/ctxhelper/stream.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ctxhelper
+
+import (
+	"context"
+	"sync"
+
+	"github.com/erda-project/erda/internal/apps/ai-proxy/vars"
+	"github.com/erda-project/erda/pkg/reverseproxy"
+)
+
+func GetIsStream(ctx context.Context) (bool, bool) {
```

Comment:
why return 2 bool？ only use 1 bool in context.

Reply:
Yes, the first bool means the value of `isStream`, and the second bool is used to confirm whether it exists or not in context.
Other Get/Put method under `ctxhelper` pkg follow the same rule.

Reply:
OK